datanode/gateway/graphql/schema.graphql

```diff
@@ -6650,3 +6670,73 @@ type EstimatedTransferFee {
   "Discount applied to the fee."
   discount: String!
 }
+
+"Individual participating in a game and their metrics"
+type IndividualGameEntity {
+  "Party ID of the individual"
+  individual: ID!
```

Comment:
Wouldn’t it be more consistent to have this as a party rather than an individual?

Reply:
I just realised you were referring to the field name rather than the documentation. In that case, the field naming I feel should be individual as it corresponds with the entity scope of the game participants, that being Individual or Team.

I have however updated the documentation to reflect that when we refer to individual, we mean an individual party, as opposed to a team and it's members.